fbchat/send.py

```diff
@@ -0,0 +1,143 @@
+# -*- coding: UTF-8 -*-
+
+from __future__ import unicode_literals
+from .listener import Listener
+from .models import Size
+
+
+class Send(Listener):
+    """Enables sending various messages to threads
+
+    Every method in this class, except `send`, `on_message` and `upload`, are
+    shortcuts of the aforementioned
+    """
+
+    #: Contains a list of all messages sent by the client
+    sent_messages = None
+
+    def send(self, thread, message):
```

Comment:
I'd like to be able to use `self.send(thread, message)` in `on_message`, but in a way where it's fully functional, that means sending images, videos, attachments and so on back. This means that `send` should be able to parse all `Message` objects, and send a proper response.